ui/apps/platform/cypress/integration/policies/policiesTable.test.js

```diff
@@ -1,15 +1,39 @@
-import * as api from '../../../constants/apiEndpoints';
-import { selectors, url } from '../../../constants/PoliciesPagePatternFly';
-import withAuth from '../../../helpers/basicAuth';
-import { generateNameWithDate } from '../../../helpers/formHelpers';
+import * as api from '../../constants/apiEndpoints';
+import { selectors, url } from '../../constants/PoliciesPagePatternFly';
+import withAuth from '../../helpers/basicAuth';
+import { generateNameWithDate } from '../../helpers/formHelpers';
 import {
     doPolicyRowAction,
     searchPolicies,
     visitPolicies,
     visitPoliciesCallback,
     visitPoliciesFromLeftNav,
-} from '../../../helpers/policiesPatternFly';
-import navSelectors from '../../../selectors/navigation';
+} from '../../helpers/policiesPatternFly';
+import navSelectors from '../../selectors/navigation';
+
+describe('Policy Management URL redirect', () => {
+    withAuth();
+
+    it('should redirect old policies URL to new policy management URL', () => {
+        cy.intercept('GET', `${api.policies.policies}?query=`).as('getPolicies');
+        cy.visit('main/policies');
```

Comment:
FYI for any future changes to test files: Replace `cy.visit(…)` with `visit(…)` from helpers/visit.js

Added in #1457 it waits on featureflags and mypermissions requests to prevent intermittent test failures